docs/user-guide/concepts.md

```diff
@@ -114,11 +114,11 @@ import (
 )
 
 func New() *VPPAgent {
-	// Prepare KVDB sync plugin with ETCD as a connector
-	etcdDataSync := kvdbsync.NewPlugin(kvdbsync.UseKV(&etcd.DefaultPlugin))
+	// Prepare kvDB sync plugin with ETCD as a connector
+	etcdDataSync := kvdbsync.NewPlugin(kvdbsync.Usekv(&etcd.DefaultPlugin))
 	
-	// Put the KVDB sync to a list of proto watchers 
-	watchers := datasync.KVProtoWatchers{
+	// Put the kvDB sync to a list of proto watchers 
+	watchers := datasync.kvProtoWatchers{
```

Comment:
@chrismetz09 This code would not even compile.. don't change the code... and IMHO, I think `KV` is better than `kv` since it's an abbrevation of `key-value`.

Reply:
Was attempting to standardize on some term for key-value and quick scan of changes missed this. Am okay on "KV" for "kv" but need to standardize terms where it is embedded:
KVDB vs kvdb vs KVdb; 
kvdb sync vs kvdbsync vs KVDB sync;